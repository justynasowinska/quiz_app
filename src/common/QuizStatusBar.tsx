import * as React from 'react';
import { StatusBar } from 'react-native';

export const QuizStatusBar = () => <StatusBar barStyle="dark-content" animated={true} />;