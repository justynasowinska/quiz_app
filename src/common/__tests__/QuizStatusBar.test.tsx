import * as React from 'react';
import { shallow } from 'enzyme';

import { QuizStatusBar } from '../QuizStatusBar';

describe('QuizStatusBar Component', () => {
    it('matches to the snapshot', () => {
        const component = shallow(<QuizStatusBar />);

        expect(component).toMatchSnapshot();
    });
});
