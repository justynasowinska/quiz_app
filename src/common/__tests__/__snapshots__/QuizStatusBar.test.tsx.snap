// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuizStatusBar Component matches to the snapshot 1`] = `
<StatusBar
  animated={true}
  barStyle="dark-content"
  showHideTransition="fade"
/>
`;
